src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1631,6 +1665,29 @@ components:
         - electoralRollNumber
         - deliveryAddressType
 
+    UpdateAnonymousElectorDocumentRequest:
+      title: UpdateAnonymousElectorDocumentRequest
+      description: Request body to update specific details on one or more Anonymous Elector Documents (AEDs). Note that explicitly setting any of the values to an empty string ("") indicates it should be removed. Otherwise, omitting the field means it will not be updated.
+      type: object
+      properties:
+        sourceReference:
+          type: string
+          example: 1f0f76a9a66f438b9bb33070
+          description: Reference in the source application that this Anonymous Elector Document is for.
+        email:
+          type: string
+          maxLength: 1024
+          format: email
+          description: The applicant's email address. This will only be updated if a non-null value is provided. Note that it will be removed if an empty string ("") is sent (if applicable).
+          example: fred.blogs@some-domain.co.uk
```

Comment:
Unsure if we ever want to *update* the email or phone to be blank?

Reply:
I need to check the validity of this use case, but in theory an elector could want us to remove the email/phone number we have on record.

Reply:
I've removed this as I think it's unlikely or very rare. If absolutely necessary, a support request could be raised, or the ERO could update the phone number to something like 0777777777!